### PR TITLE
Edit models

### PR DESCRIPTION
**Pull request by Zkat02**
Я изменила модели: user -> customer, добавила и удалила некоторые поля

### CODE REVIEW

File: website/settings.py [hunk 1]
```diff
@@ -73,14 +74,20 @@
...
+        'NAME': 'online_store',
+        'USER': 'postgres',
+        'PASSWORD': 'root',
+        'HOST': 'localhost',
+        'PORT': '5432',
     }
 }
 
 
+
```
Comment (Mariannah47) [inline website/settings.py:94]: did you mean to have 3 empty line here or you added it by accident?

File: shop/migrations/0001_initial.py [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("created_at", models.DateTimeField(auto_now_add=True)),
+            ],
+        ),
```
Comment (Mariannah47) [inline shop/migrations/0001_initial.py]: Can you please split this file the way we have 1 migration per 1 model? It is hard to review when we have a mix of everything in one file.

File: shop/admin.py [hunk 3]
```diff
@@ -1,3 +1,75 @@
...
+class Seller_Admin(admin.ModelAdmin):
+    list_display = ('user', 'seller_name', 'address')
+    list_display_links = ('user',)
+    search_fields = ('user', 'seller_name', 'address')
+
+
+admin.site.register(Seller, Seller_Admin)
+
+
+class Category_Admin(admin.ModelAdmin):
```
Comment (Mariannah47) [inline shop/admin.py]: I think you can use `@admin.register(Category)`
`from django.contrib import admin`

File: README [hunk 4]
```diff
@@ -1 +1 @@
-Веб-приложение интернет-магазина
\ No newline at end of file
+Online store
```
Comment (Mariannah47) [inline README:1]: this file is missing an empty line in the end.
ditto for `.gitignore`

**Comment by Zkat02** [comment]
> Change models: user -> customer, добавила и удалила некоторые поля

**Pull request closed without merging**